### ISSUE TITLE
Definition of indefinite integral

### ISSUE THREAD

**Issue by sean-fitzpatrick** [APEXCalculus/APEXCalculusPTX#208]
We define the indefinite integral to be the set of all antiderivatives of a function.

But [later on](https://opentext.uleth.ca/apex-standard/sec_antider.html#p-4309), we write that `\int f(x) dx = F(x) +C`, where F(x) is an antiderivative and C is a constant.

This isn't consistent. For consistency, we should write `\int f(x) dx = \{F(x)+C | C\in \mathbb{R}\}` but of course nobody does this.

What's the right way to fix this? Maybe we don't definite the indefinite integral as a set, but rather as a "general antiderivative"?
Stewart is no better on this; he says the indefinite integral represents a function, or a family of functions. (Either one, depending on how you feel, I guess?)

Another option is to state that the indefinite integral is not really an object, but a notation:

`\int f(x) dx = F(x)` is short-form for saying, "an antiderivative of f(x) is F(x)," or "the derivative of F(x) is f(x)".

**Comment by Alex-Jordan** [comment]
I think I would want to tell students that in this context, `F(x) + C` is a special shorthand notation for a set. Can we do that?

**Comment by sean-fitzpatrick** [comment]
I think we can do this, but I guess the main question is, how to say something that is consistent, but still makes sense for students?

I guess when we get to solving IVPs, we have an undetermined constant, but this time, it's *an* antiderivative, rather than an indefinite integral.

**Comment by APEXCalculus** [comment]
I think we can "be consistent" if we acknowledge that we are abusing
notation.

Building on Alex's idea, we could write
"Using Definition 5.1.2, we will write that \int f(x) dx = F(x) + C, where
we slightly abuse notation by letting ``F(x) + C'' represent a set."

I'd also like to redo the graphic decomposing the integral notation. I
think it would be wise to add a multiplication symbol between f(x) and dx.
Students tend to think that the integral symbol and the dx act as bookends.

**Comment by sean-fitzpatrick** [comment]
I've made a few changes that I hope to make available soon.

First, I want to note that I am not responsible for the comment in the source that says: "Todo: fix this weird image."
But I did try to fix the image a bit:
- apparently the syntax `right of=node` is deprecated, and should be replaced by `right=of node`. Why do `of` and `=` need to trade places? I don't know. But I swapped them.
- I added a node distance option to tighten it up a bit
- I added an extra node with a `\cdot` to emphasize the multiplication

I added a paragraph above the image pointing out that `F(x)+C` is abuse of notation, and that it really stands for a set.

I also added a paragraph below the image pointing out that another way to interpret this is that `f(x) dx` is the differential of `F(x)`, or in other words, `F'(x)=f(x)`, which is exactly what we mean when we say that `F(x)` is an antiderivative of `f(x)`.

We can work on the wording once I make the pull request.
